### ISSUE TITLE
Make Panoptes ArcFinder Demo

### ISSUE THREAD

**Issue by drphilmarshall** [drphilmarshall/SpaceWarps#225]
Things we need to do or get:

* [ ] Example test and sim gri PNG images for ArcFinder search ( @anupreeta27 )
* [ ] Manifest with feedback messages
* [ ] Classification interface with feedback messages
* [ ] Make a few people "gold standard" classifiers so we can test the creation of the dud set!

Also:
* [ ] Talk to Marten about Nero (online SWAP) status
* [ ] Ask Grant about launch/episode strategies
* [ ] Find out latest on project timelines from Chris L

Then:
* [ ] Identify which features are still missing

**Comment by aprajita** [comment]
Ignore the order of the checklist, and the last thing will not be a "then" but done as I build.

it's missing a build second demo site which i am ~ 60% through.

**Comment by aprajita** [comment]
First issue on updates solved by Cam 
https://github.com/zooniverse/Panoptes/issues/1477

SW essential things
- speed: 
need to remove confirmation screen AV to issue
-

**Comment by aprajita** [comment]
@anupreeta27 @drphilmarshall I have uploaded Anu's test data
- no obvious way to upload feedback info 
- no obvious way to implement feedback in project builder
   - [ ] raise issue AV
- no obvious way t provide different types of training subjects in 'gold mode'
   - [ ] discuss with Marten AV
- images need to resized
   - [ ] issue pan & zoom features query AV
   - [ ] resize pngs AM
- settings may need tweaking (data will be of lower depth so this is best case scenario)
   - [ ] later, fix feedback/NERO/p&z features first

**Comment by aprajita** [comment]
please refer to github issue page to see my full checklist of tasks (your email notification probably only contains the first line) from 5 days ago

**Comment by anupreeta27** [comment]
@aprajita could you say why we need to resize the pngs? is it just for some test purposes or do you mean in general for the AF candidates?

**Comment by aprajita** [comment]
see screen-grab could be larger
![interface1](https://cloud.githubusercontent.com/assets/1917970/11208291/3ca7e85c-8d12-11e5-8199-7bb820bd1952.png)

**Comment by drphilmarshall** [comment]
Agreed, larger images are important. I guess we could make just interpolate our images so that they are 400x400 *screen* pixels large, but it seems a bit wasteful.

Thanks for checking the feedback messages - I thought they were possible to define, with the training data but obviously not! I'm pretty sure that's something on the Zooniverse to-do list.

For me, the whole question/marker/button interface to the right of the image needs work. When I hit "Done" I expect to get to the next image, it's annoying (and slow) having to click again. And before this, the "Marker" button looks distracting to me. If the task only involves doing one thing, there's no need (and it's better not) to show the button that selects it! Would you agree, PI's?

**Comment by aprajita** [comment]
Yeah, I hadn't thought of the marker button itself as i just click on the image anyway, but will add it to the list of issues
we certainly can't remove it in the current way the project builder works

i've already asked about how to get rid of the confirmation page and Cam was telling me there's mixed opinions in the zoo team about whether this should appear or not.

i'll get on the other front-end things, including the feedback messages so will also ask about that.

**Comment by aprajita** [comment]
update
ChrisL said that optionally removing the confirmation page was on their to-do list
also can now ingest gold standard data with feedback. the classifications need to be in the following form. Anu, can you please produce something like this for a handful of test gold standard data?

Gold standard classificaitons should be in the form:

[{
  "links": {
    "subjects": ["123"]
  },
  "annotations": [{
    task: "T1",
    value: 3
  }, {
    task: "T2",
    value: "The value"
  }, ...]
}, ...]

**Comment by aprajita** [comment]
Recent updates

- added Tutorial but now this can't be subject based so we can't do the walk through classification :-(
tutorial text still needs to be modified

- Field guide added but no content up loaded yet - it's a 1 line perobject format so it's not how we used to have things. we also need to find a way to have the detailed SG available too

- i will need to check about whether we can still have the pop-up messages if users dont login

- there's still much to be done on the layout, image display, marker button and losing the confirmaion screen 

- we should think about the different version of subjects we want to upload and whether we want thumbnails to appear around the main image.

**Comment by aprajita** [comment]
More updates, spoke to Grant, Ali & Cam today, updates in bold

SW desirable/essential things for the interface/FrontEnd
- tutorial: 
  - [ ] no current means of including this (essential) AV to follow-up possibilities for this - **tutorial implemented but does not allow interactivity, subjects aren't linked to the tutorial, can only upload snapshots, current tutorial text uploaded without images
  - [ ] add images & modify text accordingly**
- speed: 
  - [ ] need to remove confirmation screen (essential) AV to issue https://github.com/zooniverse/Panoptes-Front-End/issues/1902 **- is possible to remove this, will ask grant/cam to implement this for us, becomes two buttons, done & done+talk instead of conf screen **
  - [ ] image load (desirable, needs actual sample data to do speed checks before any query started) - **need to upload more data**
- queue up next image:
  - [ ] was nice in SW1&2 but not really necessary **- this is not a feature of panoptes will not happen & i think it's not really important**
- pan & zoom
  - [ ] was useful particularly for blind search (desirable) check with sample blind & targeted data **- there is active discussion and ChrisL is not in favour of it. Grant will discuss with him but I am of the feeling given the DES blind images I've seen from Anu it will not be possible to do SW without zoom feature for targeted and pan & zoon for blind. Please comment on this.**
- testing feedback
  - [ ] more to add when proper example & gold standard data ingested (essential) **- we should provide a manifest with x,y position, ffeedback messages for success and non-success and they will take care of putting it in the right format or telling us how to adapt the subject metadata if that's the best way**
  - [ ] **still need to think about best means of indeitfying the different types of Gold standara data - should this be in the metdata - doesnt hurt to put this in along with success and miss feedback messages as well as providing the manifest. at least the type will be needed for NERO to work i think.**
- displaying colour settings & loss of quick dashboard
  - [ ] investigate how different colours will be shown (different versions accommodated in image upload)
  - [ ] investigate how subtracted/unsubtracted images will be shown (different versions accommodated in image upload)
  - [ ] investigate possibilities of postage stamps being shown other than play button
  **- [ ] only options are - play button or side by side (one or the other not both) - see https://www.zooniverse.org/projects/klmasters/hampshire-landscape-watch/classify**
  - changing image classification threshold to NERO input
  - [ ] AV liaise with Marten
**- back button (not there any more - remove this)
** 
Other things
- data ingestion outside Panoptes Project Builder (currently limited to 10,000 files)
   - [ ] AV to investigate best means of uploading data **Cam will help with upload, no preferred format just make it available and he will upload**


**Image sizes
- [ ] they dont have any feature to automatically scale the images and they leave this up to the browser so we'll have to produced scaled images for display.
